app/models/miq_report_result.rb

```diff
@@ -350,6 +350,14 @@ def self.with_current_user_groups
     current_user.admin_user? ? all : where(:miq_group_id => current_user.miq_group_ids)
   end
 
+  def self.with_chargeback
+    where(:db => Chargeback::SUBCLASSES)
```

Comment:
Yeah, we need to eager load the subclasses before we see them.  @Fryguy @bdunne how do we handle this for provider loading of subclasses.  I recall we eager load the subclasses, but I don't remember and git grep isn't finding it.